### ISSUE TITLE
Documentation how to run OpenSearch with security plugin enabled

### ISSUE THREAD

**Issue by kodjo-anipah** [Graylog2/graylog2-server#14432]
### Context
[HS-1318525368](https://app.hubspot.com/contacts/8267556/ticket/1318525368/)

In our [documentation](https://go2docs.graylog.org/5-0/setting_up_graylog/securing_opensearch.htm) we currently don't provide an example on how to secure OpenSearch. 

### Expected
More information on how to configure OpenSearch security plugin. 
E.g. which Default action groups action group is sufficient, or which permissions are necessary

**Comment by bernd** [comment]
Please rope in the PS team so we can determine what permissions the Graylog user in OpenSearch needs. (e.g. our anomaly detection)

**Comment by BBruce-Graylog** [comment]
@bernd Has this been addressed/documented?

**Comment by bernd** [comment]
> @bernd Has this been addressed/documented?

I don't think so. Our documentation about securing OpenSearch is still quite minimal. I don't know when @Graylog2/search has scheduled this task.

**Comment by kmerz** [comment]
Since it is a documentation issue. I opened up a ticket for documentation.

**Comment by baonq-me** [comment]
I have this issue too because I have to share OpenSearch with other applications due to cost/maintenance issues. Currently, I have to give Graylog the super admin account of my OpenSearch cluster.
